packages/jest-mock/src/index.js

```diff
@@ -26,12 +26,15 @@ export type MockFunctionMetadata = {
  */
 type MockFunctionResult = {
   /**
-   * True if the function threw.
-   * False if the function returned.
+   * Indicates the type of call result.
+   * A value of 'incomplete' indicates that the call has not completed yet.
+   * This is possible if you read the mock function result from within
+   * the mock function itself (or a function called by the mock function).
    */
-  isThrow: boolean,
+  type: 'return' | 'throw' | 'incomplete',
```

Comment:
Is there any benefit to defining a type alias in flow? In TypeScript, I would have defined:
```ts
export type MockFunctionResultType = 'return' | 'throw' | 'incomplete';
```
Then defined the "type" property as:
```ts
type: MockFunctionResultType;
```

Reply:
Dumb question; same benefit applies. I think I'll do the same here.